src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  const semicolon = ';';
+  const colon = ':';
+
+  const convertedToObject = sourceString
+    .split(semicolon)
+    .reduce((styleAccumulator, declaration) => {
+      const [property, value] = declaration.trim().split(colon);
+
+      if (value) {
+        styleAccumulator[property.trim()] = value.trim();
+      }
+
+      return styleAccumulator;
+    }, styleObject);
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation to improve readability. Align the `.split` and `.reduce` methods accordingly.